### ISSUE TITLE
Dialog tabs wrap

### ISSUE THREAD

**Issue by wildone** [websight-io/starter#143]
Is it possible to have tabs wrap or provide a way to scroll them then there are more than neatly fit?

<img width="489" alt="image" src="https://user-images.githubusercontent.com/170312/223611483-58b47e2a-34b1-4a71-aae1-6e056e3d9383.png">

**Comment by paweljozwicki** [comment]
There's no way to do this with dialog fields provided by us, but in WebSight CMS Version: 1.10.0 there are tooltips on those tabs. I'm not sure if it will meet your requirements. If not, then you should be able to provide your own dialog fields, but we have to provide some documentation on how to do this.

**Comment by wildone** [comment]
Yeah its a bit ugly looking atm with all those elipses, means you can only add so many tabs before its not friendly for authors.

AEM has a good example on this ill try digging it up. Maximise and Columns for dialogs would make big difference to Authors as well.

Tabs in large

<img width="848" alt="image" src="https://user-images.githubusercontent.com/170312/223900850-362f3293-6465-4ce9-9803-4edd4c29e85f.png">

Tabs in small

<img width="529" alt="image" src="https://user-images.githubusercontent.com/170312/223900929-c4f651d8-6ae6-4607-9c80-c7b0481ff985.png">

Dialog Maximise

<img width="577" alt="image" src="https://user-images.githubusercontent.com/170312/223901017-e2266016-b54c-4de9-95c7-4b2625e6ac74.png">

**Comment by wildone** [comment]
Also adding a help button next to the close button in the dialog so that we can define a helpurl so that we can connect help file with dialogs.

**Comment by paweljozwicki** [comment]
In fact, those are insights that should improve the authoring experience. We will have to take a closer look at them in the design session.

**Comment by paweljozwicki** [comment]
I apologize, I missed the question. No, there is no option to collapse this panel. It was a design decision not to allow for that.
